doc/classes/ARVRServer.xml

```diff
@@ -29,11 +20,16 @@
 			</argument>
 			<description>
 				This is a really important function to understand correctly. AR and VR platforms all handle positioning slightly differently.
-				For platforms that do not offer spatial tracking our origin point (0,0,0) is the location of our HMD but you have little control over the direction the player is facing in the real world.
-				For platforms that do offer spatial tracking our origin point depends very much on the system. For OpenVR our origin point is usually the center of the tracking space, on the ground. For other platforms its often the location of the tracking camera.
-				This method allows you to center our tracker on the location of the HMD, it will take the current location of the HMD and use that to adjust all our tracking data in essence realigning the real world to your players current position in your game world.
-				For this method to produce usable results tracking information should be available and this often takes a few frames after starting your game.
-				You should call this method after a few seconds have passed, when the user requests a realignment of the display holding a designated button on a controller for a short period of time, and when implementing a teleport mechanism.
+
```

Comment:
IDE doing some magic? The indentation should be kept, and the empty newlines should be removed.

Reply:
Yeah, all I did was run the doctool to update the files

Reply:
Yeah it was broken at that time, fixed since. Could you amend the commit to fix that?